Use revert-taskit branch of mongotalk

tinchodias opened the pull request:
See: https://github.com/pharo-nosql/mongotalk/pull/101

tinchodias committed:
Use revert-taskit branch of mongotalk

tinchodias commented:
@noha Do you think this rollback of taskit and use ad-hoc Process forks is worth a try?
(I mean if it runs well... this weekend I could see what are these errors on CI).

noha commented:
I think it is worth solving the problem. I was unable to find the culprit in a short time frame so I'm open to many trials. There is too much going wrong for me with the new code. You did not answer the question if the voyage and mongo code is completely changed or if the old behaviour can still be activated

tinchodias commented:
@noha: 

> if the voyage and mongo code is completely changed or if the old behaviour can still be activated"

Right now, I don't remember when you asked it, but in any case. IIUC you ask me if VOMongoRepository could have like a "beSDAM" that switches to the MongoClient and else uses the previous system. My answer is that it's not direct change but may be possible, of course. When I moved to MongoClient, I also removed instance variables in VOMongoRepository and removed the VOMongoPool (not sure if was the exact name).

estebanlm commented:
all this is very old
